docs/my-website/docusaurus.config.js

```diff
@@ -90,6 +87,15 @@ const config = {
     ({
       // Replace with your project's social card
       image: 'img/docusaurus-social-card.png',
+      algolia: {
+        // The application ID provided by Algolia
+        appId: 'NU85Y4NU0B',
+  
+        // Public API key: it is safe to commit it
+        apiKey: '4e0cf8c3020d0c876ad9174cea5c01fb',
+  
```

Comment:
looks like this is common practice : https://github.com/Clarifai/docs/blob/55b281e72952c259dcfb8795b2fb466d9255fab9/docusaurus.config.js#L54